src/Symfony/Component/Form/Tests/Fixtures/Descriptor/form_type_option_3.json

```diff
@@ -0,0 +1,6 @@
+{
+    "required": false,
+    "default": null,
+    "has_default_closures": true,
```

Comment:
I think I would keep `lazy/is_lazy` here, as it's more meant for programmatic usage rather than debug/discovering. WDYT?

Reply:
Changed to `is_lazy`.